daemon/delete.go

```diff
@@ -6,18 +6,9 @@ import (
 	"path"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/engine"
 )
 
-func (daemon *Daemon) ContainerRm(job *engine.Job) error {
-	if len(job.Args) != 1 {
-		return fmt.Errorf("Not enough arguments. Usage: %s CONTAINER\n", job.Name)
-	}
-	name := job.Args[0]
-	removeVolume := job.GetenvBool("removeVolume")
-	removeLink := job.GetenvBool("removeLink")
-	forceRemove := job.GetenvBool("forceRemove")
-
+func (daemon *Daemon) ContainerRm(name string, forceRemove, removeVolume, removeLink bool) error {
```

Comment:
I'm ok with it, I've followed your discussion on irc about 5+ args then struct so I thought it would make sense not to do a struct.